service-api/module/Application/src/Model/Entity/CaseData.php

```diff
@@ -89,7 +89,7 @@ class CaseData implements JsonSerializable
     public ?array $idMethodIncludingNation = [];
 
     #[Annotation\Required(false)]
-    public ?string $progressPage = null;
+    public ?array $progressPage = null;
```

Comment:
It would be preferable to define this as an object rather than an unshaped array, like we have for `$counterService`